pom.xml

```diff
@@ -63,7 +63,7 @@
         <guava.version>23.6.1-jre</guava.version>
         <jsr305.version>3.0.2</jsr305.version>
         <immutables.version>2.8.3</immutables.version>
-        <com.sap.cloud.lm.sl.open.version>1.89.0</com.sap.cloud.lm.sl.open.version>
+        <com.sap.cloud.lm.sl.open.version>1.90.0-SNAPSHOT</com.sap.cloud.lm.sl.open.version>
```

Comment:
Wouldn't this be better off in its own commit?

Reply:
Done